packages/anoncreds/src/formats/AnonCredsCredentialFormatService.ts

```diff
@@ -567,7 +567,7 @@ export class AnonCredsCredentialFormatService implements CredentialFormatService
     // if the proposal has an attachment Id use that, otherwise the generated id of the formats object
     const format = new CredentialFormatSpec({
       attachmentId: attachmentId,
-      format: ANONCREDS_CREDENTIAL,
+      format: ANONCREDS_CREDENTIAL_OFFER,
```

Comment:
I wanted to make sure this won't break previous versions, but we basically allow any of the identifiers for all messags, which is a bit weird but in this case helps a lot (but probably also why we didn't dettect this for so long)

Reply:
yep makes sense. i'm not sure if you guys had been interop testing with acapy, but they also have some minor issues with the attachment naming which i imagine would cause interop issues (see https://github.com/openwallet-foundation/acapy/pull/3374). Hopefully after that we can have some good interop. this is specifically coming up as i've been testing out cheqd anoncreds (being introduced to acapy soon).